lib/puma/request.rb

```diff
@@ -302,11 +300,12 @@ def fast_write_response(socket, body, io_buffer, chunked, content_length)
       else
         # for array bodies, flush io_buffer to socket when size is greater than
         # IO_BUFFER_LEN_MAX
+        size_to_first_byte = @size_to_first_byte
```

Comment:
Why the assignment? Can't we just use the instance variable in the if statements?

Reply:
Yeah, I'll change...